nucliadb_utils/nucliadb_utils/nats.py

```diff
@@ -47,6 +47,44 @@ def get_traced_jetstream(nc: NATSClient, service_name: str) -> JetStreamContext:
     return jetstream
 
 
+class message_progress_updater:
```

Comment:
PEP8 (no reason to hide the fact that this is a class even if it's a context manager)